src/index.js

```diff
@@ -72,20 +72,22 @@ export async function showLocation (options) {
       url += `&q=${title ? `${encodedTitle}&address=${encodedTitle}` : 'Location'}`
       break
     case 'google-maps':
-      url = prefixes['google-maps']
+      url = prefixes['google-maps'];
 
       if (options.googleForceLatLon && title) {
-        url += `?q=loc:${lat},+${lng}+(${encodedTitle})`
+        url += `?q=loc:${lat},+${lng}+(${encodedTitle})`;
       } else if (title) {
-        url += `?q=${encodedTitle}`
+        url += `?q=${encodedTitle}`;
+      } else if (useSourceDestiny) {
+        url += isIOS ? `/dir/'${sourceLatLng}'/'${latlng}'` : `&saddr=${sourceLatLng}&daddr=${latlng}`;
       } else {
-        url += `?q=${latlng}`
+        url += isIOS ? `/place/'${latlng}'` : `&ll=${latlng}`;
```

Comment:
Shouldn't that check be `isIOS && options.alwaysIncludeGoogle`?

Same for line 82. 

Or I could be mistaken :)

Reply:
It doesn't really matter, but yeah, it'll be a better approach